mapper.html

```diff
@@ -0,0 +1,177 @@
+<!--
+Copyright 2022 Google LLC.
+Copyright (c) Microsoft Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>BiDi-CDP Mapper</title>
+    <style>
+      body {
+        font-family: Roboto, serif;
+        font-size: 13px;
+        color: #202124;
+      }
+
+      .log {
+        padding: 12px;
+        margin-top: 16px;
+        font-family: Menlo, serif;
+        font-size: 11px;
+        line-height: 180%;
+        background: #f1f3f4;
+        border-radius: 4px;
+
+        display: flex;
+        flex-direction: column;
+        gap: 10px;
+      }
+
+      .pre {
+        overflow-wrap: break-word;
+      }
+
+      .card {
+        margin: 60px auto;
+        padding: 2px 0;
+        max-width: 900px;
+        box-shadow: 0 1px 4px rgba(0, 0, 0, 0.15), 0 1px 6px rgba(0, 0, 0, 0.2);
+        border-radius: 8px;
+      }
+
+      .divider {
+        height: 1px;
+        background: #f0f0f0;
+      }
+
+      .item {
+        padding: 16px 20px;
+      }
+
+      .text_1 {
+        font-size: 18px;
+      }
+
+      .text_2 {
+        padding-top: 5px;
+        color: #606367;
+      }
+    </style>
+  </head>
+  <div class="card">
+    <div class="item">
+      <div class="text_1">BiDi-CDP Mapper is controlling this tab</div>
+      <div class="text_2">
+        Closing or reloading it will stop the BiDi process.
+        <a
+          target="_blank"
+          title="BiDi-CDP Mapper GitHub Repository"
+          href="https://github.com/GoogleChromeLabs/chromium-bidi"
+          >Details.</a
+        >
+      </div>
+    </div>
+    <div class="divider"></div>
+    <details id="details">
+      <summary class="item">Debug information</summary>
+      <div class="divider"></div>
+      <div class="item">
+        <div>System</div>
+        <div id="system_log" class="log">
+          <div class="pre">
+            current target ID: 44BF71AB4281C0707C9F25CDD93C8422
```

Comment:
Where does this magic string come from? Should we document it?

Reply:
It was an example page. I removed it in favour of the string const in the code.